app/services/dashboard/mentor.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+module Dashboard
+  class Mentor
+    attr_reader :induction_record, :participant_profile
+
+    def initialize(induction_record:, participant_profile: induction_record&.participant_profile)
```

Comment:
Curious to know when these could not belong together? i.e. the induction_record.participant_profile is not equal to participant_profile ?  If this is not the case then could we remove participant_profile as a param?

Reply:
Oh this is legacy from an initial implementation.
I can remove the default value because now this method is meant to be called with either the induction record or the participant profile.